solutions/sapo-14-genghis.mdx

```diff
@@ -0,0 +1,244 @@
+---
+id: sapo-14-genghis
+source: SAPO 2014
+title: The Stables of Genghis Khan
+author: Kevin Sheng
+---
+
+## Solution
+
+Let's define $\texttt{lowestOps}[a][b]$ as the minimum number of operations to see if each horse is in the stables or not, given that we only consider horses in the interval $(\texttt{stableHorses[a]}, \texttt{stableHorses[b]})$, with $\texttt{stableHorses}$ being a sorted list of all the horses that are in the stables.
+
+Why use an open interval? It's to account for the horses with ID's that aren't contained in the range of the ID's of the horses of the stable. For example, if we had a horse with ID $4$, and the horses in the stables were $[6, 7, 10]$, then there would be no way for $\texttt{lowestOps}$ to account for that horse.
```

Comment:
kinda doesnt flow, can u adjust ordering a bit.

Reply:
wdym

Reply:
i think it can be at the end of editorial with a info block or as a note tbh.

Reply:
i feel like the open interval is a pretty important part of the implementation
unless you can do one w/ a closed interval

Reply:
maybe surround the open interval explanation with a warning component

Reply:
why would that be a warning though

Reply:
implementation details are usually warnings (just because there isn't a better component) 

you can phrase it like "warning! note that we're using an open interval because ..."

Reply:
so the reason is just "this is a warning because everywhere else stuff like this is a warning"?
just asking, i'm willing to do it